SETUP/CHANGELOG.md

```diff
@@ -7,13 +7,23 @@ see the git history.
 [R202009](https://github.com/DistributedProofreaders/dproofreaders/releases/tag/R202009)
 first before upgrading to R202102 or later releases.**
 
-## R??????
+## R202409
 Scripts supporting this upgrade are in `SETUP/upgrade/21`
 
 **This is the last release to support PHP 7.4. Future releases will only
-support PHP 8.0 and later.**
+support PHP 8.1 and later.**
```

Comment:
Rationale behind this:
* PHP 8.0 is already out of support.
* PHP 8.1 is in Ubuntu 22.04 which is still supported. It also includes some useful [type features](https://www.php.net/manual/en/language.types.declarations.php). We tested with PHP 8.1 on the clone system earlier this year and know it works.
* PHP 8.3 is what will be running on PROD after the upgrade later this year.